UT4MasterServer.Web/src/pages/Servers/data/get-servers-test-response.ts

```diff
@@ -0,0 +1,2033 @@
+// Sample response from matchMakingRequest for testing
+export const getServersTestResponse = [
```

Comment:
you can use example from from before epic shut down servers. it's located in `OldReferenceCode/HubMenuResponse.json`. It's longer by 1000 lines.

Reply:
That's good to know that it is there, but I'd rather have the response from the new master server than the old.